sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/WhereLogic.cs

```diff
@@ -45,7 +43,7 @@ public WhereQuery Where(string field, QueryComparisonOperator comparisonOperator
         /// <param name="containOperator"> ADT contains operator defined by the ADT query language. </param>
         /// <param name="searched"> Field of possible options to check for the 'value' parameter. </param>
         /// <returns> ADT query that already contains SELECT and FROM. </returns>
-        public WhereQuery Where(string value, QueryContainsOperator containOperator, string[] searched)
+        public WhereLogic Contains(string value, QueryContainsOperator containOperator, string[] searched)
```

Comment:
So this one ... I would separate Contains and NotContains methods to separate ones and get rid of the QueryContainsOperator ... the more I look at it the more I realize it's vague and counter-intuitive. You call Contains method but in the parameters you can change it to NotContains ...